### PR TITLE
Docker tests

### PR DESCRIPTION
**Pull request by vtsutsman**

### CODE REVIEW

**Comment by coffeine-009** [comment]
Can one of the admins verify this patch?

**Comment by coffeine-009** [comment]
Looks good for me.

File: src/test/java/com/coffeine/virtuoso/music/controller/FunctionalSuite.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+/**
+ * Suite case for functional tests.
+ */
+@RunWith( Suite.class )
+@Suite.SuiteClasses({
+    FunctionalSongControllerTest.class,
```
Comment (vtsutsman) [inline src/test/java/com/coffeine/virtuoso/music/controller/FunctionalSuite.java]: Good point.

**Comment by coffeine-009** [comment]
trigger build.

File: src/test/java/com/coffeine/virtuoso/music/controller/FunctionalSongControllerTest.java [hunk 2]
```diff
@@ -56,7 +57,7 @@ public void testListActionSuccess() throws Exception {
         this.mockMvc.perform(
             get( "/music/songs?page={page}&limit={limit}", 1, 10 )
             .header( "Authorization", this.session.getAuthorizationHeader() )
-        )
+        ).andDo(print())
             .andExpect( status().isOk() )
```
Comment (coffeine-009) [inline src/test/java/com/coffeine/virtuoso/music/controller/FunctionalSongControllerTest.java]: Do not forget to remove this.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,12 @@ Beginners can easily learn favorite songs.
 
 ## Getting Started
 ### On the server
+#### Docker
+```bash
+docker build --rm -t coffeine/virtuoso-api .
+docker run -p 8080:8080 -d coffeine/virtuoso-api
+docker run -it coffeine/virtuoso-api bash
+```
 _(Coming soon)_
```
Comment (vtsutsman) [inline README.md]: Correct please.

**Pull request closed without merging**